templates/page.html.twig

```diff
@@ -76,33 +76,36 @@
 
   {{ page.help }}
 
-  {% if page.page_title %}
-    {% block page_title %}
-      <div class="cd-page-title cd-container">
-        {{ page.page_title }}
-      </div>
-    {% endblock %}
-  {% endif %}
-
   {% block main %}
     {# Link to skip to the main content is in html.html.twig #}
     <main role="main" {{ attributes.setAttribute('id', 'main-content').addClass(layout_classes) }}>
 
-      {% if page.sidebar_first %}
-        {# We are using CSS pseudo class :empty to hide these aside elements if the region prints empty.
-          This means we cannot have any white space between tags (as below).
-          Attn: Having the twig debug enabled means the div is no longer empty.
-          See Drupal issues/953034 for description and current status.
-        #}
-        <aside class="cd-layout-sidebar-first" role="complementary">{{ page.sidebar_first }}</aside>
+      {% if page.page_title %}
+        {% block page_title %}
+          <div class="cd-page-title">
```

Comment:
@orakili I notice the wrapper div `cd-page-title` doesn't seem to get rendered. It's not important as there are no rules associated with it any more, but I'm curious as to why exactly.